InnerEye/ML/deep_learning_config.py

```diff
@@ -295,9 +295,7 @@ class DeepLearningConfig(GenericConfig, CudaAwareConfig):
                          "to that set.",
                      allow_None=True)
     perform_training_set_inference: bool = \
-        param.Boolean(False,
-                      doc="If False (default), run full image inference on validation and test set after training. If "
-                          "True, also run full image inference on the training set")
+        param.Boolean(False, doc="If True, run full image inference on the training set at the end of training.")
```

Comment:
Re-extend. I did not like the previous version because it implied that this flag=False means inference on val/test automatically but in reality it is controlled by the next flag.